examples/ring_app/ring-example.conf

```diff
@@ -30,6 +30,19 @@ authorization: {
             allow: "$1"
             name: "users allowed by backreference"
             sort-order: 500
-        }
+        },
+        {
+            match-request: {
+                path: "/hello/extension-allowed"
+                type: "path"
+            }
+            allow: {
+                extensions: {
+                    my_extension_shortname: "extensions value"
+                }
+            }
+            name: "users allowed by backreference"
```

Comment:
I think this will blow up since the names for each rule need to be unique.  Maybe make this "users allowed by matching extension" or something like that?